src/command_modules/azure-cli-interactive/azure/cli/command_modules/interactive/tests/test_query_injection.py

```diff
@@ -26,13 +26,12 @@ class QueryInjection(unittest.TestCase):
     """ tests using the query gesture for the interactive mode """
     def __init__(self, *args, **kwargs):
         super(QueryInjection, self).__init__(*args, **kwargs)
-        from azclishell.app import Shell, validate_contains_query
+        from azclishell.app import Shell
```

Comment:
Do you test the undefined scenario:
`?? stuff --param ??`

What does interactive do in that case?

Reply:
so in this case, interactive will inject whatever the json result from the last command was in each of the two locations. Since this will be either a json object/list and is not a valid command, this will throw a error

Reply:
Sure it will throw an error... something like `az: error: argument _command_package: invalid choice: '{a:b}'`

That's not terribly helpful. Interactive is the "hold your hand" approach to the CLI, so I strongly recommend checking for this scenario and giving a specific, actionable error message.